all/entrypoint.sh

```diff
@@ -71,6 +74,13 @@ while test $# -gt 0; do
             GEN_GATEWAY=true
             shift
             ;;
+        --lint)
+            LINT=true
+            if [ "$#" -gt 1 ] && [[ $2 != -* ]]; then
+                LINT_CHECKS=$2
```

Comment:
`shift` here too (i.e. consume the $2), otherwise everything looks good.

I agree - the bash is getting kinda overboard, it's on my short-list to port this to a more sane language.